common/proxy/proxy_test.go

```diff
@@ -19,6 +19,8 @@ package proxy
 
 import (
 	"context"
+	hessian "github.com/apache/dubbo-go-hessian2"
```

Comment:
OMG. move it to the second import block, pls.

Reply:
done